### DISCUSSION TITLE
My achievements are not shown

### DISCUSSION THREAD

**Discussion by aduffner** [community/community#56706] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, 
the URL [https://github.com/aduffner?tab=achievements](https://github.com/aduffner?tab=achievements) gives me a 404 (which was the solution at similar discussions) and my settings are also fine. Maybe a bug? To be fair i should have only one achievement: "Pull shark", due the multiple merges a week ago. I have never seen it on my profile. Are there other settings/circumstances i should be aware of?

Best regards
Alex

**Comment by aduffner** [comment]
Done everything which was mentioned in #37577.

**Comment by NyanKaungSet** [comment, marked as answer]
Hello @aduffner, 

This issue is kinda common in GitHub. See #50552 and #56662.

Since the Achievements are still in Beta mode, there maybe some issues and bugs. It is possible that the achievements aren't shown in some users.

Try getting other achievements like Yolo and Quickdraw.

If your achievements aren't still showing, try contacting the [GitHub Support](https://support.github.com/). They may be able to help you out.
<br>
Hope this helps,

**Reply by aduffner** [reply]
Thank you for the explanation 😅

**Reply by NyanKaungSet** [reply]
It was my pleasure.
